src/components/ThirdPartyCollectionDetailPage/ThirdPartyCollectionDetailPage.container.ts

```diff
@@ -0,0 +1,36 @@
+import { connect } from 'react-redux'
+import { push } from 'connected-react-router'
+import { isLoadingType } from 'decentraland-dapps/dist/modules/loading/selectors'
+import { getData as getWallet } from 'decentraland-dapps/dist/modules/wallet/selectors'
+import { RootState } from 'modules/common/types'
+import { getThirdPartyCollectionId } from 'modules/location/selectors'
+import { getCollection, isOnSaleLoading, getLoading as getLoadingCollection } from 'modules/collection/selectors'
+import { getCollectionItems, getLoading as getLoadingItem } from 'modules/item/selectors'
+import { FETCH_COLLECTIONS_REQUEST, DELETE_COLLECTION_REQUEST } from 'modules/collection/actions'
+import { openModal } from 'modules/modal/actions'
+import { FETCH_ITEMS_REQUEST } from 'modules/item/actions'
+import { MapStateProps, MapDispatchProps, MapDispatch } from './ThirdPartyCollectionDetailPage.types'
+import CollectionDetailPage from './ThirdPartyCollectionDetailPage'
+
+const mapState = (state: RootState): MapStateProps => {
+  const collectionId = getThirdPartyCollectionId(state) || ''
+  const collection = getCollection(state, collectionId)
+
+  return {
+    wallet: getWallet(state)!,
+    collection,
+    items: getCollectionItems(state, collectionId),
+    isOnSaleLoading: isOnSaleLoading(state),
```

Comment:
I don't find this variable to be in use, what is it for?

Reply:
Not used! it was for the on sale toggle, good eye